frameworks.rst

```diff
@@ -93,11 +93,11 @@ links!
 `weblayer <http://packages.python.org/weblayer>`_
     weblayer is a lightweight, componentised package for writing WSGI
     applications.
-`Zope 3 <http://www.zope.org/>`_
-    The venerable Python web framework, recreated anew in Zope 3, and
-    now a WSGI application.  It *seems* to have some WSGI bits deep
-    inside the publisher, but they aren't really documented at this
-    time.
+`Zope <https://zope.readthedocs.io>`_
+    The former Zope 2 web application server is now nearing version 4.
```

Comment:
this description is confusing. This doesn't tell which version it is. Thats probably because it's the description copy pasted from Zope2 zope.org website

This is the newer version, the description should say that it's the newer version, and what advantages zope 4-5 bring compared to zope 2-3.

Reply:
Zope 4 (and 5) are newer versions of what used to be Zope 2. Zope 3 is a different project, but it is completely dead and has been for many years. There is no point in mentioning Zope 3, doing so will just add confusion.